mk/spksrc.tc-vers.mk

```diff
@@ -64,4 +64,5 @@ endif
 
 ifeq ($(TC_VERS),7.0)
 TC_BUILD = 40000
+TC_VERS = 7.0.0
```

Comment:
We might get unexpected side effects, as the DSM version is still 7.0 and not 7.0.0